tasks/main.yml

```
@@ -66,7 +77,6 @@
 
 - name: "importing virtualenv tasks"
   import_tasks: virtualenv.yml
```

Let's only use include the virtualenv.yml tasks when it's necessary. This will allow us to not have to specify `when` on each generic task.
```suggestion
  import_tasks: virtualenv.yml
  when: ansible_os_family in ['RedHat', 'Darwin']
```